src/components/assets/Asset.vue

```diff
@@ -50,6 +50,7 @@ export default {
     getAsset(this.$route.params.id).then(data => {
       this.asset = data
       this.$ready(this.asset.title)
+      location.hash = `#asset:${this.asset.id}`
```

Comment:
Does this need to be `window.parent.location.hash` when in iframe?

Reply:
@johncrossman I think maybe it does? SuiteC's never had to deal with being in-or-out of iframes before, but the front end might need an "am I in an iframe?" util method like Junction has:

https://github.com/ets-berkeley-edu/junction/blob/master/src/mixins/Iframe.vue#L39

Reply:
@johncrossman Oh, that's annoying - it may not be possible to set the parent's hash cross-domain. Current SuiteC has to do it via fancy cross-frame messaging: https://github.com/ets-berkeley-edu/suitec/blob/master/public/app/shared/utilService.js#L349